Package.swift

```diff
@@ -21,6 +21,7 @@ let package = Package(
         .target(
           name: "KeychainAccess",
           path: "Lib/KeychainAccess",
+          exclude: ["PrivacyInfo.xcprivacy"],
```

Comment:
Why is the manifest excluded instead of adding it as resource to the package? As the manifest should be part of the package.
```
resources: [
                .copy("PrivacyInfo.xcprivacy")
            ],
```